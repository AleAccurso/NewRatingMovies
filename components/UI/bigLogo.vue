<template>
  <v-container class="grey lighten-5 mainLogo" id="logo">
    <img :src="logoURL" alt="Rating Movies Logo" />
  </v-container>
</template>

<script>
export default {
  data(){
    return {
      logoURL: "@/assets/ratingMoviesLogo.png"
    }
  }
}
</script>

<style>
#logo{
    display:block;
    margin: 30px 0;
    padding: 0;
    text-align: center;
}
</style>
