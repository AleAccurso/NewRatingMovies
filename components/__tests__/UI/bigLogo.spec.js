import { shallowMount } from "@vue/test-utils";
import bigLogo from "../../UI/bigLogo";

let wrapper = shallowMount(bigLogo);

describe("bigLogo", () => {
  let logo = wrapper.find(".mainLogo");

  test("Render correctly", () => {
    expect(logo.exists()).toBe(true);
  });
});
