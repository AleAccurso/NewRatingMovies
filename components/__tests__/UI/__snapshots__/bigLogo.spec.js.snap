// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bigLogo Render correctly 1`] = `
<v-container
  class="grey lighten-5 mainLogo"
  id="logo"
>
  <img
    alt="Rating Movies Logo"
    src="@/assets/ratingMoviesLogo.png"
  />
</v-container>
`;
